.changes/unreleased/Features-20230829-152412.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Features
+body: Support limiting get_catalog by object name
+time: 2023-08-29T15:24:12.649104-04:00
+custom:
+  Author: mikealfare
+  Issue: "4997"
```

Comment:
Can we link to an issue in the dbt-snowflake repo so that the link in the changelog will work?

Otherwise, it will link to a page that doesn't exist.

Reply:
[Here](https://github.com/dbt-labs/dbt-snowflake/blob/1.6.latest/CHANGELOG.md#under-the-hood-1) is an example from the dbt-snowflake 1.6 changelog that doesn't work when linked to a dbt-core issue:

> test foreign key + unique constraint rendering ([#7417](https://github.com/dbt-labs/dbt-snowflake/issues/7417))

Reply:
Fair point. I'll update that to this PR number (which actually works because GH doesn't care if something is an issue or a PR).